conf/jasminegraph-server.properties

```diff
@@ -70,6 +74,9 @@ org.jasminegraph.fl.aggregator=true
 org.jasminegraph.fl.org.training=false
 org.jasminegraph.fl.organization.file=/home/ubuntu/software/jasminegraph/conf/hosts.txt
 
```

Comment:
The config key `org.jasminegraph.fl.local` already holds the same value. Why do we introduce `org.jasminegraph.fl.location.local` here? Should `org.jasminegraph.fl.location.local` and `org.jasminegraph.fl.local` be two config keys or one?

Reply:
Removed org.jasminegraph.fl.location.local configuration as it would only be required for training

Reply:
Ok